http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -602,13 +602,25 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
 
       deleteDiskAction = msg.diskIdToDelete.traverse { diskId =>
         for {
-          _ <- deleteDiskResource(Left(runtime.id), msg.workspaceId, auth)
+          runtimeRecordOpt <- controlledResourceQuery
+            .getWsmRecordForRuntime(msg.runtimeId, WsmResourceType.AzureDisk)
+            .transaction
+          runtimeRecord <- F.fromOption(
+            runtimeRecordOpt,
+            AzureDiskResourceDeletionError(Left(msg.runtimeId),
+                                           msg.workspaceId,
+                                           "No disk resource found for delete azure disk. No-op for wsmDao.deleteDisk."
```

Comment:
thinking about in a production environment ...
this error will get raised. 

some questions I am asking myself:
- who will be handling the exception?
- if no one is catching it to handle the exception (it will be breaking out of the various processing and downstream execution of this method) - what is the intended result of this error?

which leads me to be more curious about:
- if we are really in a situation where (for some reason) we are being asked to delete a disk and the disk does not exist, should we instead clean up our own state and log (or metrics?)?